software/cncjs.yml

```
@@ -1,6 +1,6 @@
 name: CNCjs
 website_url: https://cnc.js.org/
-description: A web-based interface for CNC milling controller running Grbl, Smoothieware, or TinyG.
+description: CNC milling controller running Grbl, Smoothieware, or TinyG.
```

```suggestion
description: Web interface for CNC milling controllers running Grbl, Smoothieware, or TinyG.
```

Specifying "Web interface" is actually correct here, since the controller is the hardware component.